core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -587,6 +589,11 @@ public int registerOrForward(String subject,
                                boolean normalize,
                                Map<String, String> headerProperties)
       throws SchemaRegistryException {
+
+    if(QualifiedSubject.create(this.tenant(), subject).getSubject().equals(GLOBAL_RESOURCE_NAME)){
+      throw new OperationNotPermittedException(GLOBAL_RESOURCE_NAME + " subject name is not allowed");
```

Comment:
It would probably be better to move this code to `SubjectVersionsResource.register` and then throw `Errors.invalidSubjectException(subjectName)`

Reply:
Also for consistency, we may want this check in `ConfigResource.updateSubjectLevelConfig` and `ModeResource.updateMode`

Reply:
Okay, will make these changes. :)